config/.env

```diff
@@ -0,0 +1,8 @@
+# env variables for local development
+# do not include sensitive data here
+
+NODE_ENV="development"
+API_URL="https://api-dev.creativecommons.engineering"
+
+# feature flags
+WATERMARK=true
```

Comment:
@aldenstpage the env variables for the feature flags will be in this file. This file here is just for local development, but server env variables during build time in CI will be used instead of these